tower-blocking-service/src/lib.rs

```diff
@@ -0,0 +1,26 @@
+use tokio::runtime::current_thread::Runtime;
+use tower_service::Service;
+use tower_service_util::BoxService;
+
+pub struct BlockingSvc<Request, Response, Err> {
```

Comment:
The name `BlockingService` made me think that it was a wrapper that took a "sync service" and made it async via a threadpool strategy.

I'm not sure what name we could use for the other way around though.

Reply:
I was considering `SyncService` but that might conflict weirdly with `UnsyncBoxService`.